.github/workflows/Build.yml

```diff
@@ -46,7 +46,7 @@ jobs:
         sql-versions: ['mariadb', 'postgresql', 'sqlite']
         mode: ['', '--no-dev']
 
-    name: PHP${{ matrix.php-versions }} - ${{ matrix.sql-versions }} - ${{ matrix.mode }}
+    name: PHP${{ matrix.php-versions }} - ${{ matrix.sql-versions }} ${{ matrix.mode }}
```

Comment:
Github workflow files are out of my wheelhouse. Could you explain what this change does and why it did not work before? I am too lazy to read the Workflow Docs.

Reply:
OK, agreed. It looks better.